buildpack.toml

```diff
@@ -139,21 +139,7 @@ api = "0.7"
     name = "JAVA_TOOL_OPTIONS"
 
   [[metadata.dependencies]]
-    cpes = ["cpe:2.3:a:oracle:jdk:17.0.12:*:*:*:*:*:*:*"]
-    id = "jdk"
-    name = "Oracle JDK"
-    purl = "pkg:generic/oracle-jdk@17.0.12?arch=amd64"
-    sha256 = "311f1448312ecab391fe2a1b2ac140d6e1c7aea6fbf08416b466a58874f2b40f"
-    stacks = ["*"]
-    uri = "https://download.oracle.com/java/17/archive/jdk-17.0.12_linux-x64_bin.tar.gz"
-    version = "17.0.12"
-
-    [[metadata.dependencies.licenses]]
-      type = "Oracle No-Fee Terms and Conditions License"
-      uri = "https://www.oracle.com/downloads/licenses/no-fee-license.html"
-
-  [[metadata.dependencies]]
-    cpes = ["cpe:2.3:a:oracle:jdk:21.0.5:*:*:*:*:*:*:*"]
+    cpes = ["cpe:2.3:a:oracle:jdk:21.0.4:*:*:*:*:*:*:*"]
```

Comment:
Why `21.0.5` to `21.0.4`? Or is the diff tricking me?

Reply:
```suggestion
    cpes = ["cpe:2.3:a:oracle:jdk:21.0.5:*:*:*:*:*:*:*"]
```

Good catch! Thanks